### PR TITLE
task 6,7,8 added

### PR DESCRIPTION
**Pull request by nidhi-tagline**

### CODE REVIEW

File: task6.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+string = 'PQRQRQRQ'
+substring = 'QRQ'
+print(f'String: {string} \nSubstring: {substring}\n')
+
+str_list = []
+for i in range(len(string)):
+    str_list.append(string[i:i+len(substring)])
+    if i == len(string)-len(substring):
+        break
+
```
Comment (himanshu-suthar-tagline) [inline task6.py]: Please use regex.

File: task8.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# sort list
+
+numbers = [2, 5, 6, 1, 3, 8, 9, 10]
+
+for i in range(len(numbers)):
+    for j in range(len(numbers) - 1):
+        if numbers[j] > numbers[j+1]:
+            temp = numbers[j]
+            numbers[j] = numbers[j+1]
+            numbers[j+1] = temp
```
Comment (himanshu-suthar-tagline) [inline task8.py:10]: Tell me which sorting algorithm you have used here.

Comment (nidhi-tagline) [reply task8.py:10]: It is bubble sort.
1. outer loop is for iteration
2. inner loop is for swapping which check if num[j] is greater than num[j+1] and swap elements if condition is true else move to next element
3. After completion of one loop i will increment and repeat process until list is sorted.

File: task6.py (hunk 1 above)
Comment (nidhi-tagline) [reply task6.py]: ok, sir.

**Comment by nidhi-tagline** [comment]
task 6 changed

**Pull request merged**